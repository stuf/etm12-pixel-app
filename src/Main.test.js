import * as React from 'karet';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Index from './Main';

it('matches snapshot', () => {
  const wrap = shallow(<Index />);
  expect(toJson(wrap)).toMatchSnapshot();
});
