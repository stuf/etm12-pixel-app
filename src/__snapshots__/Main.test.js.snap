// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<main
  className="root"
>
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        path="/editor"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </BrowserRouter>
</main>
`;
